constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -50,32 +57,7 @@ public ConstructorIO(String apiToken, String apiKey, boolean isHTTPS, String hos
         } else {
             this.protocol = "http";
         }
-        this.encoder = new URLEncodedUtils();
-        Unirest.setDefaultHeader("Content-Type", "application/json");
-    }
-
-    /**
-     * Serializes url params in a rudimentary way.
-     *
-     * Unlike in the other Constructor.io clients, this one only takes in hashmaps,
-     * and you must write other helper methods to serialize other things.
-     *
-     * @param params HashMap of the parameters to encode.
-     * @return The encoded parameters, as a String.
-     */
-    public static String serializeParams(HashMap<String, String> params) throws UnsupportedEncodingException {
-        String urlString = "";
-        boolean isFirst = true;
-        for (String key : params.keySet()) {
-            if (!isFirst) {
-                urlString += "&";
-            }
-            urlString += URLEncoder.encode(key, "UTF-8");
-            urlString += "=";
-            urlString += URLEncoder.encode(params.get(key), "UTF-8");
-            isFirst = false;
-        }
-        return urlString;
+        this.credentials = "Basic " + Base64.getEncoder().encodeToString((this.apiToken + ":").getBytes());
```

Comment:
Out of curiosity, why encode to base64?

Reply:
Oh also why add the colon?

Reply:
https://tools.ietf.org/html/rfc2617#section-2

Reply:
oh.